app/services/geocoding.js

```diff
@@ -6,10 +6,17 @@ const mapboxGeocoding = require('@mapbox/mapbox-sdk/services/geocoding');
 const baseClient = mapboxClient({ accessToken: Config.MAPBOX_ACCESS_TOKEN });
 const geocodingService = mapboxGeocoding(baseClient);
 
-export const geocodeCityInput = async ({ City, State = '' }) => geocodingService.forwardGeocode({
-  query: `${City} ${State}`,
-  limit: 1,
-}).send();
+export const geocodeCityInput = async ({ City, State = '' }) => {
+  // the geocoder does not respond well to `d. c.` as the state
+  if (State === 'd. c.') {
```

Comment:
That works, except it's rare enough for people to say "district of columbia" that it's nice to fix the case where people say DC.

I think there's probably a better fix for this, but it would take time to investigate.